### PR TITLE
Add support for contentFiles to nuget package manger

### PR DESCRIPTION
**Pull request by dannyvv**
and use it to produce the cloudbuild private package

### CODE REVIEW

File: Public/Sdk/Public/Managed/Tools/NuGet/pack.dsc [hunk 1]
```diff
@@ -93,6 +96,19 @@ export interface Dependency {
     version: string;
 }
 
+export interface ContentFile {
+    /** The location of the file or files to include, subject to exclusions specified by the exclude attribute. The path is relative to the .nuspec file unless an absolute path is specified. The wildcard character * is allowed, and the double wildcard ** implies a recursive folder search. */
+    include: string,
```
Comment (smera) [inline Public/Sdk/Public/Managed/Tools/NuGet/pack.dsc]: >string [](start = 13, length = 6)

Maybe doing something more strongly typed?

Comment (smera) [reply Public/Sdk/Public/Managed/Tools/NuGet/pack.dsc]: Same for the one below

---
In reply to: [279451820](https://github.com/Microsoft/BuildXL/pull/214#discussion_r279451820) [](ancestors = 279451820)

Comment (dannyvv) [reply Public/Sdk/Public/Managed/Tools/NuGet/pack.dsc]: This is just for initial experimentation. Nuget has it's own custom string mathing rules here, so opting to keep generic for now.

**Comment by dannyvv** [comment]
Tested in CloudBuild repo with this package and files are properly copied.

**Pull request merged**